### ISSUE TITLE
Ensure minimum Charge information in prices series (Feature-AC4e)

### ISSUE THREAD

**Issue by prtandrup** [Energinet-DataHub/geh-charges#1203]
Dependency:
Refactored input validation in Charge Information flow, as the way input validation is done there, should be reused here.

Note:
`effectiveDate` is not mandatory in CIM schema of NotifyChangeOfPriceList, but it is considered mandatory for DataHub3 and thus must be present when sending out Charge Price Notifications (D08)

- [x] (4e) Given a charge **prices** update request is received with the mandatory master data fields for identification (Charge ID, Owner, Type and "effectiveDate" as well)
When processing it
Then other fields are ignored
And outbound notifications will only include mandatory master data fields for charge identification, incl. effectivedate

**Comment by prtandrup** [comment]
Yesterday, I tested the submission of a D08 price series with only the mandatory information being present (owner, type, charge id and effective date). This was blocked by validation rule `ChargeResolutionCanNotBeUpdatedRule` as it is passed internally as Resolution.Unknown, making it appear as an attempt to change it.

I suspect a similar challenge exists for `VatClassificationValidationRule` and `ChangingTariffTaxValueNotAllowedRule`

**Comment by HenrikSommer** [comment]
Some work is already done, see [PR 1420](https://github.com/Energinet-DataHub/geh-charges/pull/1420)

**Comment by prtandrup** [comment]
Note: the price notifications sent out, contains only the required master data. See dump below from a test today. So that part of the AC is OK.

![image.png](https://images.zenhubusercontent.com/60ae42d0916bb92d8b000901/06a0dcd3-4c0b-4179-8be5-cd0bfc28faa6)

**Comment by HenrikSommer** [comment]
@prtandrup's comment of Jun 10 is no longer relevant, due to the separation of ChargePrice from the ChargeInformation flow.

**Comment by prtandrup** [comment]
### Tested in T-001 :heavy_check_mark:

```
AC:
Given a charge prices update request is received with the mandatory master data fields for identification (Charge ID, Owner, Type and "effectiveDate" as well)
When processing it
Then other fields are ignored
And outbound notifications will also only include mandatory master data fields for charge identification
```

**Submitted a Charge Price Request (D08) with a complete set of charge information values (for tax tariff with ID: SyoTar0004)**

Request:
````
<?xml version="1.0" encoding="UTF-8"?>
<cim:RequestChangeOfPriceList_MarketDocument xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cim="urn:ediel.org:structure:requestchangeofpricelist:0:1" xsi:schemaLocation="urn:ediel.org:structure:requestchangeofpricelist:0:1 urn-ediel-org-structure-requestchangeofpricelist-0-1.xsd">
    <cim:mRID>DocId2022-09-15T06:21:37.628Z</cim:mRID>
    <cim:type>D10</cim:type>
    <cim:process.processType>D08</cim:process.processType>
    <cim:businessSector.type>23</cim:businessSector.type>
    <cim:sender_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:sender_MarketParticipant.mRID>
    <cim:sender_MarketParticipant.marketRole.type>EZ</cim:sender_MarketParticipant.marketRole.type>
    <cim:receiver_MarketParticipant.mRID codingScheme="A10">5790001330552</cim:receiver_MarketParticipant.mRID>
    <cim:receiver_MarketParticipant.marketRole.type>DDZ</cim:receiver_MarketParticipant.marketRole.type>
    <cim:createdDateTime>2022-09-15T06:21:37Z</cim:createdDateTime>
    <cim:MktActivityRecord>
        <cim:mRID>OpId2022-09-15T06:21:37.628Z</cim:mRID>
        <cim:ChargeGroup>
            <cim:ChargeType>
                <cim:chargeTypeOwner_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:chargeTypeOwner_MarketParticipant.mRID>
                <cim:type>D03</cim:type>
                <cim:mRID>SyoTar0004</cim:mRID>
                <cim:name>Charge name</cim:name>
                <cim:description>The charge description</cim:description>
                <cim:priceTimeFrame_Period.resolution>P1M</cim:priceTimeFrame_Period.resolution>
                <cim:effectiveDate>2022-09-30T22:00:00Z</cim:effectiveDate>
                <cim:VATPayer>D01</cim:VATPayer>
                <cim:transparentInvoicing>false</cim:transparentInvoicing>
                <cim:taxIndicator>false</cim:taxIndicator>
                <cim:Series_Period>
                    <cim:resolution>PT1H</cim:resolution>
                    <cim:timeInterval>
                        <cim:start>2022-09-30T22:00Z</cim:start>
                        <cim:end>2022-10-01T22:00Z</cim:end>
                    </cim:timeInterval>
                    <cim:Point>
                        <cim:position>1</cim:position>
                        <cim:price.amount>1.789</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>2</cim:position>
                        <cim:price.amount>2.002</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>3</cim:position>
                        <cim:price.amount>3.003</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>4</cim:position>
                        <cim:price.amount>4.004</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>5</cim:position>
                        <cim:price.amount>5.005</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>6</cim:position>
                        <cim:price.amount>6.006</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>7</cim:position>
                        <cim:price.amount>7.007</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>8</cim:position>
                        <cim:price.amount>8.008</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>9</cim:position>
                        <cim:price.amount>9.009</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>10</cim:position>
                        <cim:price.amount>10.010</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>11</cim:position>
                        <cim:price.amount>11.011</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>12</cim:position>
                        <cim:price.amount>12.012</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>13</cim:position>
                        <cim:price.amount>13.013</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>14</cim:position>
                        <cim:price.amount>14.014</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>15</cim:position>
                        <cim:price.amount>15.015</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>16</cim:position>
                        <cim:price.amount>16.016</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>17</cim:position>
                        <cim:price.amount>17.017</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>18</cim:position>
                        <cim:price.amount>18.018</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>19</cim:position>
                        <cim:price.amount>19.019</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>20</cim:position>
                        <cim:price.amount>20.020</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>21</cim:position>
                        <cim:price.amount>21.021</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>22</cim:position>
                        <cim:price.amount>22.022</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>23</cim:position>
                        <cim:price.amount>23.023</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>24</cim:position>
                        <cim:price.amount>24.024</cim:price.amount>
                    </cim:Point>
                </cim:Series_Period>
            </cim:ChargeType>
        </cim:ChargeGroup>
    </cim:MktActivityRecord>
</cim:RequestChangeOfPriceList_MarketDocument>
````

**Grid Access Provider received notification:** 

Note that the following information fields are not present, although they were provided as input:
````
* <cim:name>
* <cim:description>
* <cim:priceTimeFrame_Period.resolution>
* <cim:VATPayer>
* <cim:transparentInvoicing>
* <cim:taxIndicator>
````
````
<?xml version="1.0" encoding="utf-8"?>
<cim:NotifyPriceList_MarketDocument xmlns:cim="urn:ediel.org:structure:notifypricelist:0:1">
    <cim:mRID>cd30c67c771449668e4621e8d555ab33</cim:mRID>
    <cim:type>D12</cim:type>
    <cim:process.processType>D08</cim:process.processType>
    <cim:businessSector.type>23</cim:businessSector.type>
    <cim:sender_MarketParticipant.mRID codingScheme="A10">5790001330552</cim:sender_MarketParticipant.mRID>
    <cim:sender_MarketParticipant.marketRole.type>DDZ</cim:sender_MarketParticipant.marketRole.type>
    <cim:receiver_MarketParticipant.mRID codingScheme="A10">8100000000016</cim:receiver_MarketParticipant.mRID>
    <cim:receiver_MarketParticipant.marketRole.type>DDM</cim:receiver_MarketParticipant.marketRole.type>
    <cim:createdDateTime>2022-09-15T06:21:48Z</cim:createdDateTime>
    <cim:MktActivityRecord>
        <cim:mRID>04b2fe31bcd64d3eac82cc5ce9560654</cim:mRID>
        <cim:snapshot_DateAndOrTime.dateTime>2022-09-15T06:21:40Z</cim:snapshot_DateAndOrTime.dateTime>
        <cim:ChargeGroup>
            <cim:ChargeType>
                <cim:chargeTypeOwner_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:chargeTypeOwner_MarketParticipant.mRID>
                <cim:type>D03</cim:type>
                <cim:mRID>SyoTar0004</cim:mRID>
                <cim:effectiveDate>2022-09-30T22:00:00Z</cim:effectiveDate>
                <cim:Series_Period>
                    <cim:resolution>PT1H</cim:resolution>
                    <cim:timeInterval>
                        <cim:start>2022-09-30T22:00Z</cim:start>
                        <cim:end>2022-10-01T22:00Z</cim:end>
                    </cim:timeInterval>
                    <cim:Point>
                        <cim:position>1</cim:position>
                        <cim:price.amount>1.789000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>2</cim:position>
                        <cim:price.amount>2.002000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>3</cim:position>
                        <cim:price.amount>3.003000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>4</cim:position>
                        <cim:price.amount>4.004000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>5</cim:position>
                        <cim:price.amount>5.005000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>6</cim:position>
                        <cim:price.amount>6.006000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>7</cim:position>
                        <cim:price.amount>7.007000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>8</cim:position>
                        <cim:price.amount>8.008000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>9</cim:position>
                        <cim:price.amount>9.009000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>10</cim:position>
                        <cim:price.amount>10.010000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>11</cim:position>
                        <cim:price.amount>11.011000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>12</cim:position>
                        <cim:price.amount>12.012000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>13</cim:position>
                        <cim:price.amount>13.013000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>14</cim:position>
                        <cim:price.amount>14.014000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>15</cim:position>
                        <cim:price.amount>15.015000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>16</cim:position>
                        <cim:price.amount>16.016000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>17</cim:position>
                        <cim:price.amount>17.017000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>18</cim:position>
                        <cim:price.amount>18.018000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>19</cim:position>
                        <cim:price.amount>19.019000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>20</cim:position>
                        <cim:price.amount>20.020000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>21</cim:position>
                        <cim:price.amount>21.021000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>22</cim:position>
                        <cim:price.amount>22.022000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>23</cim:position>
                        <cim:price.amount>23.023000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>24</cim:position>
                        <cim:price.amount>24.024000</cim:price.amount>
                    </cim:Point>
                </cim:Series_Period>
            </cim:ChargeType>
        </cim:ChargeGroup>
    </cim:MktActivityRecord>
</cim:NotifyPriceList_MarketDocument>
````

**And just as AC is a bit ambiguous, here is a Charge Prices request with mandatory fields only:**
Request
````
<?xml version="1.0" encoding="UTF-8"?>
<cim:RequestChangeOfPriceList_MarketDocument xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cim="urn:ediel.org:structure:requestchangeofpricelist:0:1" xsi:schemaLocation="urn:ediel.org:structure:requestchangeofpricelist:0:1 urn-ediel-org-structure-requestchangeofpricelist-0-1.xsd">
    <cim:mRID>DocId2022-09-15T06:25:55.351Z</cim:mRID>
    <cim:type>D10</cim:type>
    <cim:process.processType>D08</cim:process.processType>
    <cim:businessSector.type>23</cim:businessSector.type>
    <cim:sender_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:sender_MarketParticipant.mRID>
    <cim:sender_MarketParticipant.marketRole.type>EZ</cim:sender_MarketParticipant.marketRole.type>
    <cim:receiver_MarketParticipant.mRID codingScheme="A10">5790001330552</cim:receiver_MarketParticipant.mRID>
    <cim:receiver_MarketParticipant.marketRole.type>DDZ</cim:receiver_MarketParticipant.marketRole.type>
    <cim:createdDateTime>2022-09-15T06:25:55Z</cim:createdDateTime>
    <cim:MktActivityRecord>
        <cim:mRID>OpId2022-09-15T06:25:55.351Z</cim:mRID>
        <cim:ChargeGroup>
            <cim:ChargeType>
                <cim:chargeTypeOwner_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:chargeTypeOwner_MarketParticipant.mRID>
                <cim:type>D03</cim:type>
                <cim:mRID>SyoTar0004</cim:mRID>
                <cim:effectiveDate>2022-09-30T22:00:00Z</cim:effectiveDate>
                <cim:Series_Period>
                    <cim:resolution>PT1H</cim:resolution>
                    <cim:timeInterval>
                        <cim:start>2022-09-30T22:00Z</cim:start>
                        <cim:end>2022-10-01T22:00Z</cim:end>
                    </cim:timeInterval>
                    <cim:Point>
                        <cim:position>1</cim:position>
                        <cim:price.amount>1.890</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>2</cim:position>
                        <cim:price.amount>2.002</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>3</cim:position>
                        <cim:price.amount>3.003</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>4</cim:position>
                        <cim:price.amount>4.004</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>5</cim:position>
                        <cim:price.amount>5.005</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>6</cim:position>
                        <cim:price.amount>6.006</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>7</cim:position>
                        <cim:price.amount>7.007</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>8</cim:position>
                        <cim:price.amount>8.008</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>9</cim:position>
                        <cim:price.amount>9.009</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>10</cim:position>
                        <cim:price.amount>10.010</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>11</cim:position>
                        <cim:price.amount>11.011</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>12</cim:position>
                        <cim:price.amount>12.012</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>13</cim:position>
                        <cim:price.amount>13.013</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>14</cim:position>
                        <cim:price.amount>14.014</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>15</cim:position>
                        <cim:price.amount>15.015</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>16</cim:position>
                        <cim:price.amount>16.016</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>17</cim:position>
                        <cim:price.amount>17.017</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>18</cim:position>
                        <cim:price.amount>18.018</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>19</cim:position>
                        <cim:price.amount>19.019</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>20</cim:position>
                        <cim:price.amount>20.020</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>21</cim:position>
                        <cim:price.amount>21.021</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>22</cim:position>
                        <cim:price.amount>22.022</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>23</cim:position>
                        <cim:price.amount>23.023</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>24</cim:position>
                        <cim:price.amount>24.024</cim:price.amount>
                    </cim:Point>
                </cim:Series_Period>
            </cim:ChargeType>
        </cim:ChargeGroup>
    </cim:MktActivityRecord>
</cim:RequestChangeOfPriceList_MarketDocument>
````

**Notification to Grid Access Provider:**
````
<?xml version="1.0" encoding="utf-8"?>
<cim:NotifyPriceList_MarketDocument xmlns:cim="urn:ediel.org:structure:notifypricelist:0:1">
    <cim:mRID>04408fbb59e44bc09db84579403c9258</cim:mRID>
    <cim:type>D12</cim:type>
    <cim:process.processType>D08</cim:process.processType>
    <cim:businessSector.type>23</cim:businessSector.type>
    <cim:sender_MarketParticipant.mRID codingScheme="A10">5790001330552</cim:sender_MarketParticipant.mRID>
    <cim:sender_MarketParticipant.marketRole.type>DDZ</cim:sender_MarketParticipant.marketRole.type>
    <cim:receiver_MarketParticipant.mRID codingScheme="A10">8100000000016</cim:receiver_MarketParticipant.mRID>
    <cim:receiver_MarketParticipant.marketRole.type>DDM</cim:receiver_MarketParticipant.marketRole.type>
    <cim:createdDateTime>2022-09-15T06:26:20Z</cim:createdDateTime>
    <cim:MktActivityRecord>
        <cim:mRID>6bc625f05c544b94b2bc1f33cad77eae</cim:mRID>
        <cim:snapshot_DateAndOrTime.dateTime>2022-09-15T06:25:50Z</cim:snapshot_DateAndOrTime.dateTime>
        <cim:ChargeGroup>
            <cim:ChargeType>
                <cim:chargeTypeOwner_MarketParticipant.mRID codingScheme="A10">5790000432752</cim:chargeTypeOwner_MarketParticipant.mRID>
                <cim:type>D03</cim:type>
                <cim:mRID>SyoTar0004</cim:mRID>
                <cim:effectiveDate>2022-09-30T22:00:00Z</cim:effectiveDate>
                <cim:Series_Period>
                    <cim:resolution>PT1H</cim:resolution>
                    <cim:timeInterval>
                        <cim:start>2022-09-30T22:00Z</cim:start>
                        <cim:end>2022-10-01T22:00Z</cim:end>
                    </cim:timeInterval>
                    <cim:Point>
                        <cim:position>1</cim:position>
                        <cim:price.amount>1.890000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>2</cim:position>
                        <cim:price.amount>2.002000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>3</cim:position>
                        <cim:price.amount>3.003000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>4</cim:position>
                        <cim:price.amount>4.004000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>5</cim:position>
                        <cim:price.amount>5.005000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>6</cim:position>
                        <cim:price.amount>6.006000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>7</cim:position>
                        <cim:price.amount>7.007000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>8</cim:position>
                        <cim:price.amount>8.008000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>9</cim:position>
                        <cim:price.amount>9.009000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>10</cim:position>
                        <cim:price.amount>10.010000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>11</cim:position>
                        <cim:price.amount>11.011000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>12</cim:position>
                        <cim:price.amount>12.012000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>13</cim:position>
                        <cim:price.amount>13.013000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>14</cim:position>
                        <cim:price.amount>14.014000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>15</cim:position>
                        <cim:price.amount>15.015000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>16</cim:position>
                        <cim:price.amount>16.016000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>17</cim:position>
                        <cim:price.amount>17.017000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>18</cim:position>
                        <cim:price.amount>18.018000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>19</cim:position>
                        <cim:price.amount>19.019000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>20</cim:position>
                        <cim:price.amount>20.020000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>21</cim:position>
                        <cim:price.amount>21.021000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>22</cim:position>
                        <cim:price.amount>22.022000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>23</cim:position>
                        <cim:price.amount>23.023000</cim:price.amount>
                    </cim:Point>
                    <cim:Point>
                        <cim:position>24</cim:position>
                        <cim:price.amount>24.024000</cim:price.amount>
                    </cim:Point>
                </cim:Series_Period>
            </cim:ChargeType>
        </cim:ChargeGroup>
    </cim:MktActivityRecord>
</cim:NotifyPriceList_MarketDocument>
````
